core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="dot-edit-content-form" *ngIf="formData$ | async as form; else noContent">
+    <dot-edit-content-form [formData]="form" (formSubmit)="saveContent($event)" />
```

Comment:
This have no tests, if I remove the `[formData]="form"` nothing breaks in tests but the functionality will be broken.